### PR TITLE
SPR1-40: Fix Python 3, FITS and logging

### PR DESCRIPTION
**Pull request by ludwigschwardt**
- Convert Python 3 zip objects to lists to make `plot_xyz` work.
- Replace PyFITS with astropy.io.fits to make warning go away.
- Get rid of ancient PyFITS Header API in the process.
- Replace naughty `logging.basicConfig` with katdal's trick.

### CODE REVIEW

**Comment by ludwigschwardt** [comment]
For fun I checked that I can still open files and plot data from XDM, FF, KAT-7, RTS and MeerKAT 😆 

xdm_compscan_target

**Comment by ppakotze** [comment]
Ah nice, suspect this might help Tsys on Moon reduction to run again under python3.

**Comment by ludwigschwardt** [comment]
Yes, please check!

File: scape/plots_basic.py [hunk 1]
```diff
@@ -271,7 +271,8 @@ def monotonic(x):
         raise ValueError('Shape mismatch between x and y (segment lengths are %s vs %s)' %
                          ([np.shape(s)[0] for s in x], [np.shape(s)[0] for s in y]))
     if z is not None:
-        yx_shape = zip([np.shape(s)[0] for s in y], [np.shape(s)[0] for s in x])
+        yx_shape = list(zip([np.shape(s)[0] for s in y],
```
Comment (spassmoor) [inline scape/plots_basic.py:274]: I wish there was a more elegant way to do this

**Review by spassmoor — APPROVED** [review]

**Pull request merged**